android/src/main/java/com/revenuecat/purchases/react/RNPurchasesModule.java

```diff
@@ -313,8 +313,13 @@ public void invalidateCustomerInfoCache() {
     }
 
     @ReactMethod
-    public void setProxyURLString(String proxyURLString) {
-        CommonKt.setProxyURLString(proxyURLString);
+    public void setProxyURLString(String proxyURLString, Promise promise) {
+        try {
+            CommonKt.setProxyURLString(proxyURLString);
+            promise.resolve(null); // Resolve the promise with no value
+        } catch (Exception e) {
```

Comment:
This should never throw an exception, so I think it should be safe to remove this.

Reply:
Removed!